javascript/javascript1/week1/housePrice.js

```diff
@@ -0,0 +1,33 @@
+//3. Housey pricey (A house price estimator)
+
+// Peter's house parameters
+let peterHouseWidth = 8;
+let peterHouseHeight = 10;
+let peterHouseDepth = 10;
+let peterHouseGardenSize = 100;
+let peterHouseRealPrice = 2500000;
+
+// Julia's house parameters
+let juliaHouseWidth = 5;
+let juliaHouseHeigth = 8;
+let juliaHouseDepth = 11;
+let juliaHouseGardenSize = 70;
+let juliaHouseRealPrice = 1000000;
+
+let peterVolume = peterHouseWidth * peterHouseHeight * peterHouseDepth
+let juliaVolume = juliaHouseWidth * juliaHouseHeigth * juliaHouseDepth
```

Comment:
Would be more correct to call those `peterHouseVolume` and `juliaHouseVolume`.

Reply:
changed to peterHouseVolume  and juliaHouseVolume...